apps/web/package.json

```diff
@@ -33,6 +33,8 @@
     "@lexical/markdown": "^0.12.5",
     "@lexical/react": "^0.12.5",
     "@lexical/utils": "^0.12.5",
+    "@pushprotocol/restapi": "^1.4.45",
```

Comment:
why not their react sdk?

Reply:
It is their react-sdk, they have ui-components, nothing specific react sdk. Even their react example is also using same method.

Reply:
Ohh you are correct!